dnscommon.r2py

```diff
@@ -519,9 +520,25 @@ def _read_single_answer(answer_index, dns_query_data):
 
   # There are a lot of different types of queries. We can parse all of these ones 
   # in the same way, though.
-  simple_answers = ['A', 'NS', 'CNAME', 'MD', 'MB', 'MF', 'MG', 'MR', 'MX', 'PTR']
+  simple_answers = ['A', 'CNAME', 'MD', 'MB', 'MF', 'MG', 'MR']
   if answer_type in simple_answers:
     read_index, resource_data['address'] = _parse_answer_address(read_index, dns_query_data)
+  elif answer_type == 'NS':
+    read_index, resource_data['nameserver'] = _parse_address(read_index,dns_query_data)
```

Comment:
We should not combine all because when you query for MX or reverse or AAAA etc then you have additional section containing extra data. We don't need that but if we don't separate then packet dictionary will contain mixed details. So by separating, packet dictionary will contain all information with their respective query types.
for eg.
````
;; QUESTION SECTION:
;google.com.			IN	MX

;; ANSWER SECTION:
google.com.		600	IN	MX	20 alt1.aspmx.l.google.com.
google.com.		600	IN	MX	30 alt2.aspmx.l.google.com.
google.com.		600	IN	MX	40 alt3.aspmx.l.google.com.
google.com.		600	IN	MX	10 aspmx.l.google.com.
google.com.		600	IN	MX	50 alt4.aspmx.l.google.com.

;; ADDITIONAL SECTION:
alt1.aspmx.l.google.com. 156	IN	A	64.233.186.27
````
or
````
;google.com.			IN	NS

;; ANSWER SECTION:
google.com.		14400	IN	NS	ns4.google.com.
google.com.		14400	IN	NS	ns1.google.com.
google.com.		14400	IN	NS	ns2.google.com.
google.com.		14400	IN	NS	ns3.google.com.

;; ADDITIONAL SECTION:
ns1.google.com.		13328	IN	A	216.239.32.10
ns2.google.com.		13956	IN	A	216.239.34.10
ns3.google.com.		2225	IN	A	216.239.36.10
ns4.google.com.		2315	IN	A	216.239.38.10

````